README.md

```diff
@@ -748,6 +752,54 @@ The documentation on using Closure Stylesheets can be found
   `bazel run @io_bazel_rules_closure//closure/stylesheets -- --help`
 
 
+## closure\_proto\_js\_library
+
+```python
+load("@io_bazel_rules_closure//closure:defs.bzl", "closure_proto_js_library")
+closure_proto_js_library(name, srcs, add_require_for_enums, namespace_prefix,
+                         error_on_name_conflict, binary, import_style)
+```
+
+Defines a set of Protocol Buffer files.
+
+Documentation: [Protocol Buffers][protobuf] [JS][protobuf-js]
+[Generator Options](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/js/js_generator.h).
+
+#### Implicit Output Targets
+
+- *name*.js: A generated protocol buffer JavaScript library.
+
+### Arguments
+
+- **name:** ([Name][name]; required) A unique name for this rule. Convention
+  states that such rules be named `foo_proto`.
+
+- **srcs:** (List of [labels][labels]; required) A list of `.proto` source
+  files that represent this library.
+
+- **add_require_for_enums:** (Boolean; optional; default is `False`) Add a
+  `goog.requires()` call for each enum type used. If false, a forward
+  declaration with `goog.forwardDeclare` is produced instead.
+
+- **namespace_prefix:** (String; optional; default is `None`) Specify this
+  to prefix the generated namespace.
+
+- **error_on_name_conflict:** (Boolean; optional; default is `True`) Errors
+  if there are two types that would generate the same output file.
+
+- **binary:** (Boolean; optional; default is `True`) Enable binary-format support.
+
+- **import_style:** (String; optional; default is `None`) Specifies the type
```

Comment:
Changed documented default to "IMPORT_CLOSURE". In terms of the implementation which is more idiomatic bazel (None which defers to the plugins default of IMPORT_CLOSURE or an explicit IMPORT_CLOSURE which is passed to the plugin?

I'm happy naming the values with or without the "IMPORT_" prefix. My initial thought was to use the same values as the underlying tool.